catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -417,4 +417,8 @@
         <bundle>mvn:org.apache.mina/mina-core/${mina.version}</bundle>
     </feature>
 
+    <feature name="catalog-plugin-async" install="manual" version="${project.version}">
```

Comment:
This need to be changed to `install="manual"` then there is a PostProcessPlugin, right?